{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"error","message":"toifalar olishda kutilmaganda xato yuz berdi toifalar olishda kutilmaganda xato yuz berdi","stack":"Error: toifalar olishda kutilmaganda xato yuz berdi\n    at C:\\Users\\User\\FarxodDadajonov\\dars\\routes\\categories.js:8:9\n    at newFn (C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\express-async-errors\\index.js:16:20)\n    at Layer.handle [as handle_request] (C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at Route.dispatch (C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\express\\lib\\router\\route.js:112:3)\n    at newFn (C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\express-async-errors\\index.js:16:20)\n    at Layer.handle [as handle_request] (C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\express\\lib\\router\\index.js:281:22\n    at Function.process_params (C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\express\\lib\\router\\index.js:341:12)\n    at next (C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\express\\lib\\router\\index.js:275:10)"}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"error","message":"Kutilmagan xato Kutilmagan xato","stack":"Error: Kutilmagan xato\n    at Object.<anonymous> (C:\\Users\\User\\FarxodDadajonov\\dars\\index.js:26:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)\n    at node:internal/main/run_main_module:17:47"}
{"level":"error","message":"Kutilmagan xato Kutilmagan xato","stack":"Error: Kutilmagan xato\n    at Object.<anonymous> (C:\\Users\\User\\FarxodDadajonov\\dars\\index.js:29:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)\n    at node:internal/main/run_main_module:17:47"}
{"code":"ERR_UNHANDLED_REJECTION","level":"error","message":"This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason \"yana boshqa kutilmagan xato\". This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason \"yana boshqa kutilmagan xato\".","stack":"UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason \"yana boshqa kutilmagan xato\"."}
{"level":"error","message":"Kutilmagan xato Kutilmagan xato","stack":"Error: Kutilmagan xato\n    at Object.<anonymous> (C:\\Users\\User\\FarxodDadajonov\\dars\\index.js:32:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)\n    at node:internal/main/run_main_module:17:47"}
{"level":"error","message":"unhandledRejection xatosi\n"}
{"level":"error","message":"Kutilmagan xato Kutilmagan xato","stack":"Error: Kutilmagan xato\n    at Object.<anonymous> (C:\\Users\\User\\FarxodDadajonov\\dars\\index.js:32:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)\n    at node:internal/main/run_main_module:17:47"}
{"level":"error","message":"unhandledRejection xatosi\nundefined"}
{"level":"error","message":"Kutilmagan xato Kutilmagan xato","stack":"Error: Kutilmagan xato\n    at Object.<anonymous> (C:\\Users\\User\\FarxodDadajonov\\dars\\index.js:32:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)\n    at node:internal/main/run_main_module:17:47"}
{"level":"error","message":"unhandledRejection xatosi \n undefined"}
{"level":"error","message":"Kutilmagan xato Kutilmagan xato","stack":"Error: Kutilmagan xato\n    at Object.<anonymous> (C:\\Users\\User\\FarxodDadajonov\\dars\\index.js:32:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)\n    at node:internal/main/run_main_module:17:47"}
{"level":"error","message":"unhandledRejection xatosi \n undefined"}
{"level":"error","message":"Kutilmagan xato Kutilmagan xato","stack":"Error: Kutilmagan xato\n    at Object.<anonymous> (C:\\Users\\User\\FarxodDadajonov\\dars\\index.js:34:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)\n    at node:internal/main/run_main_module:17:47"}
{"level":"error","message":"unhandledRejection xatosi \n undefined"}
{"date":"Tue Feb 15 2022 20:34:33 GMT+0500 (Узбекистан, стандартное время)","error":{},"exception":true,"level":"error","message":"uncaughtException: Kutilmagan xato\nError: Kutilmagan xato\n    at Object.<anonymous> (C:\\Users\\User\\FarxodDadajonov\\dars\\index.js:39:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)\n    at node:internal/main/run_main_module:17:47","os":{"loadavg":[0,0,0],"uptime":18474},"process":{"argv":["C:\\Program Files\\nodejs\\node.exe","C:\\Users\\User\\FarxodDadajonov\\dars\\index.js"],"cwd":"C:\\Users\\User\\FarxodDadajonov\\dars","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":37632690,"external":39835357,"heapTotal":59457536,"heapUsed":37307392,"rss":74133504},"pid":7916,"uid":null,"version":"v16.11.1"},"stack":"Error: Kutilmagan xato\n    at Object.<anonymous> (C:\\Users\\User\\FarxodDadajonov\\dars\\index.js:39:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)\n    at node:internal/main/run_main_module:17:47","trace":[{"column":7,"file":"C:\\Users\\User\\FarxodDadajonov\\dars\\index.js","function":null,"line":39,"method":null,"native":false},{"column":14,"file":"node:internal/modules/cjs/loader","function":"Module._compile","line":1101,"method":"_compile","native":false},{"column":10,"file":"node:internal/modules/cjs/loader","function":"Module._extensions..js","line":1153,"method":".js","native":false},{"column":32,"file":"node:internal/modules/cjs/loader","function":"Module.load","line":981,"method":"load","native":false},{"column":12,"file":"node:internal/modules/cjs/loader","function":"Module._load","line":822,"method":"_load","native":false},{"column":12,"file":"node:internal/modules/run_main","function":"Function.executeUserEntryPoint [as runMain]","line":79,"method":"executeUserEntryPoint [as runMain]","native":false},{"column":47,"file":"node:internal/main/run_main_module","function":null,"line":17,"method":null,"native":false}]}
{"date":"Tue Feb 15 2022 20:34:49 GMT+0500 (Узбекистан, стандартное время)","error":{},"exception":true,"level":"error","message":"uncaughtException: Kutilmagan xato\nError: Kutilmagan xato\n    at Object.<anonymous> (C:\\Users\\User\\FarxodDadajonov\\dars\\index.js:39:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)\n    at node:internal/main/run_main_module:17:47","os":{"loadavg":[0,0,0],"uptime":18489},"process":{"argv":["C:\\Program Files\\nodejs\\node.exe","C:\\Users\\User\\FarxodDadajonov\\dars\\index.js"],"cwd":"C:\\Users\\User\\FarxodDadajonov\\dars","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":37520566,"external":39835357,"heapTotal":59719680,"heapUsed":36947464,"rss":73719808},"pid":3000,"uid":null,"version":"v16.11.1"},"stack":"Error: Kutilmagan xato\n    at Object.<anonymous> (C:\\Users\\User\\FarxodDadajonov\\dars\\index.js:39:7)\n    at Module._compile (node:internal/modules/cjs/loader:1101:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)\n    at node:internal/main/run_main_module:17:47","trace":[{"column":7,"file":"C:\\Users\\User\\FarxodDadajonov\\dars\\index.js","function":null,"line":39,"method":null,"native":false},{"column":14,"file":"node:internal/modules/cjs/loader","function":"Module._compile","line":1101,"method":"_compile","native":false},{"column":10,"file":"node:internal/modules/cjs/loader","function":"Module._extensions..js","line":1153,"method":".js","native":false},{"column":32,"file":"node:internal/modules/cjs/loader","function":"Module.load","line":981,"method":"load","native":false},{"column":12,"file":"node:internal/modules/cjs/loader","function":"Module._load","line":822,"method":"_load","native":false},{"column":12,"file":"node:internal/modules/run_main","function":"Function.executeUserEntryPoint [as runMain]","line":79,"method":"executeUserEntryPoint [as runMain]","native":false},{"column":47,"file":"node:internal/main/run_main_module","function":null,"line":17,"method":null,"native":false}]}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6000chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6001chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6001chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"level":"info","message":"6001chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"_message":"Category validation failed","errors":{"name":{"kind":"maxlength","message":"Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50).","name":"ValidatorError","path":"name","properties":{"maxlength":50,"message":"Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50).","path":"name","type":"maxlength","value":"ccccccccccccccccccccccccccccccccccccccccccccccccccc"},"value":"ccccccccccccccccccccccccccccccccccccccccccccccccccc"}},"level":"error","message":"Category validation failed: name: Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50). Category validation failed: name: Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50).","stack":"ValidationError: Category validation failed: name: Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50).\n    at model.Object.<anonymous>.Document.invalidate (C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\mongoose\\lib\\document.js:2962:32)\n    at C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\mongoose\\lib\\document.js:2753:17\n    at C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\mongoose\\lib\\schematype.js:1325:9\n    at processTicksAndRejections (node:internal/process/task_queues:78:11)"}
{"level":"info","message":"6001chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"_message":"Category validation failed","errors":{"name":{"kind":"maxlength","message":"Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50).","name":"ValidatorError","path":"name","properties":{"maxlength":50,"message":"Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50).","path":"name","type":"maxlength","value":"ccccccccccccccccccccccccccccccccccccccccccccccccccc"},"value":"ccccccccccccccccccccccccccccccccccccccccccccccccccc"}},"level":"error","message":"Category validation failed: name: Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50). Category validation failed: name: Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50).","stack":"ValidationError: Category validation failed: name: Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50).\n    at model.Object.<anonymous>.Document.invalidate (C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\mongoose\\lib\\document.js:2962:32)\n    at C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\mongoose\\lib\\document.js:2753:17\n    at C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\mongoose\\lib\\schematype.js:1325:9\n    at processTicksAndRejections (node:internal/process/task_queues:78:11)"}
{"level":"info","message":"6001chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"_message":"Category validation failed","errors":{"name":{"kind":"maxlength","message":"Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50).","name":"ValidatorError","path":"name","properties":{"maxlength":50,"message":"Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50).","path":"name","type":"maxlength","value":"ccccccccccccccccccccccccccccccccccccccccccccccccccc"},"value":"ccccccccccccccccccccccccccccccccccccccccccccccccccc"}},"level":"error","message":"Category validation failed: name: Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50). Category validation failed: name: Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50).","stack":"ValidationError: Category validation failed: name: Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50).\n    at model.Object.<anonymous>.Document.invalidate (C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\mongoose\\lib\\document.js:2962:32)\n    at C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\mongoose\\lib\\document.js:2753:17\n    at C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\mongoose\\lib\\schematype.js:1325:9\n    at processTicksAndRejections (node:internal/process/task_queues:78:11)"}
{"level":"info","message":"6001chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"_message":"Category validation failed","errors":{"name":{"kind":"maxlength","message":"Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50).","name":"ValidatorError","path":"name","properties":{"maxlength":50,"message":"Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50).","path":"name","type":"maxlength","value":"ccccccccccccccccccccccccccccccccccccccccccccccccccc"},"value":"ccccccccccccccccccccccccccccccccccccccccccccccccccc"}},"level":"error","message":"Category validation failed: name: Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50). Category validation failed: name: Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50).","stack":"ValidationError: Category validation failed: name: Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50).\n    at model.Object.<anonymous>.Document.invalidate (C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\mongoose\\lib\\document.js:2962:32)\n    at C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\mongoose\\lib\\document.js:2753:17\n    at C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\mongoose\\lib\\schematype.js:1325:9\n    at processTicksAndRejections (node:internal/process/task_queues:78:11)"}
{"level":"info","message":"6001chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
{"_message":"Category validation failed","errors":{"name":{"kind":"maxlength","message":"Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50).","name":"ValidatorError","path":"name","properties":{"maxlength":50,"message":"Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50).","path":"name","type":"maxlength","value":"ccccccccccccccccccccccccccccccccccccccccccccccccccc"},"value":"ccccccccccccccccccccccccccccccccccccccccccccccccccc"}},"level":"error","message":"Category validation failed: name: Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50). Category validation failed: name: Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50).","stack":"ValidationError: Category validation failed: name: Path `name` (`ccccccccccccccccccccccccccccccccccccccccccccccccccc`) is longer than the maximum allowed length (50).\n    at model.Object.<anonymous>.Document.invalidate (C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\mongoose\\lib\\document.js:2962:32)\n    at C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\mongoose\\lib\\document.js:2753:17\n    at C:\\Users\\User\\FarxodDadajonov\\dars\\node_modules\\mongoose\\lib\\schematype.js:1325:9\n    at processTicksAndRejections (node:internal/process/task_queues:78:11)"}
{"level":"info","message":"6001chi portni eshitishni boshladim..."}
{"level":"debug","message":"MongoDBga ulanish hosil qilindi..."}
